BossExpertise.cs

```diff
@@ -10,22 +10,32 @@ namespace BossExpertise
     public class BossExpertise : Mod
 	{
 
-		public static bool? FakeExpert = false;
+		public static int CurrentDifficulty = 0;
```

Comment:
I don't get it. How do we differentiate between fake difficulty for boss AI, and the real difficulty?

Reply:
Check it again